### PR TITLE
Created the GUI Internet Speed Tester

### LINKED ISSUE #289: Internet Speed Tester
### Is your feature request related to a problem? Please describe
A simple utility in **Python** to check the **internet speed of a network**.

### Describe the solution you'd like
I will be using the [**speedtest**](https://github.com/sivel/speedtest-cli) library in python.

### Describe alternatives you've considered
The utility will be return the **upload, download and ping** of a network.

### PR DESCRIPTION
**Pull request by dishant-yadav**
## Description
Created the GUI Internet Speed Tester with TKinter Library

Fixes #289 

## Dependencies
[Speed Test](https://github.com/sivel/speedtest-cli)
TKinter

## Future Improvements

Add an image of an speed meter and improve the UI of the app.

## Screenshots of relevant screens

Screenshot from 2022-10-11 00-57-00

## Developer's checklist

- [🗸 ] My PR follows the style guidelines of this project
- [ 🗸] I have performed a self-check on my work

**If changes are made in the code:**

- [🗸 ] I have followed the [coding guidelines](https://google.github.io/styleguide/jsguide.html)
- [ 🗸] My changes in code generate no new warnings
- [𐄂 ] My changes are breaking another fix/feature of the project
- [🗸 ] I have added test cases to show that my feature works
- [🗸 ] I have added relevant screenshots in my PR

### CODE REVIEW

**Comment by dishant-yadav** [comment]
@bartick Please review my PR

File: scripts/Internet Speed Tester/main.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    # speed = Speedtest()
+    # download = speed.download()
+    # upload = speed.upload()
+    # ping = speed.results.ping
+    # download_speed = round(download/(8 / 1024 / 1024),2)
+    # upload_speed = round(upload/(8 / 1024 / 1024),2)
+    # ping = round(ping, 2)
+    down_lab.config(text='Download Speed : ' + str(3.2) + " Mbps")
+    upload_lab.config(text='Upload Speed : ' + str(3.1) + " Mbps")
+    ping_lab.config(text='Ping : ' + str(46) + " ms")
```
Comment (bartick) [inline scripts/Internet Speed Tester/main.py:19]: Your code doesn't serve the purpose of checking the internet speed. These are just constant values

Comment (bartick) [inline scripts/Internet Speed Tester/main.py:19]: Your code doesn't serve the purpose of checking the internet speed. These are just constant values

**Review by bartick — CHANGES_REQUESTED** [review]

File: scripts/Internet Speed Tester/main.py (hunk 1 above)
Comment (dishant-yadav) [reply scripts/Internet Speed Tester/main.py]: Actually I was facing some issues with the **speedtest library**. So to test the app I hard coded the values. 
I am so sorry for that.
@bartick Please review the code now

Comment (dishant-yadav) [reply scripts/Internet Speed Tester/main.py]: Screenshot from 2022-10-11 22-37-47

**Review by bartick — APPROVED** [review]

**Pull request merged**